apps/condo/domains/ticket/hooks/usePollTicketComments.tsx

```diff
@@ -34,21 +34,26 @@ export function usePollTicketComments ({
 
         const now = new Date().toISOString()
         const lastSyncAt = localStorage.getItem(LOCAL_STORAGE_SYNC_KEY)
+        let newSyncedAt
 
-        const result = await refetchSyncComments({
-            where: {
-                ...pollCommentsQuery,
-                updatedAt_gt: lastSyncAt || now,
-            },
-        })
-        const ticketComments = result?.data?.ticketComments?.filter(Boolean) || []
+        try {
+            const result = await refetchSyncComments({
+                where: {
+                    ...pollCommentsQuery,
+                    updatedAt_gt: lastSyncAt || now,
+                },
+            })
+            const ticketComments = result?.data?.ticketComments?.filter(Boolean) || []
```

Comment:
thought we used get in these cases

Reply:
I agree with this. 
it doesn't look quite good, since below in another block a code structure is used using the get function

Reply:
lodash get erases types of return values, i think it's better to use ts where it's possible.
Rewrote block below to `?.` for more consistent